hmda/src/main/scala/hmda/persistence/submission/HmdaValidationError.scala

```diff
@@ -259,6 +259,7 @@ object HmdaValidationError
             .thenRun { validationState =>
               val updatedStatus =
                 SubmissionStatus.valueOf(validationState.statusCode)
+              log.info(s"Verify Quality for $submissionId")
```

Comment:
This should come after the line with `replyTo` since after that line the update would have happened and the text should be changed to `Verified Quality for <submissionId>`

Reply:
In that case, should `SignSubmission` be updated as well?

Reply:
I believe that one is fine since `publishSignEvent` returns a `Future[Done]`.